charts/prometheus-memcached-exporter/Chart.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: prometheus-memcached-exporter
+description: Prometheus exporter for Memcached metrics
+type: application
+version: 1.0.0
```

we should start with this.
```suggestion
version: 0.1.0
```

---

I've put 1.0.0 because I found that value [in docs](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#versioning). Should I still fix (and update the docs)?

---

https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase